aspnetcore/common/samples/WebApplication1/Startup.cs

```diff
@@ -75,7 +75,29 @@ public void Configure(IApplicationBuilder app, IHostingEnvironment env, ILoggerF
 
             app.UseIdentity();
 
-            // Add external authentication middleware below. To configure them please see http://go.microsoft.com/fwlink/?LinkID=532715
+            app.UseFacebookAuthentication(new FacebookOptions()
```

Comment:
Social logins don't belong in the core common/samples/WebApplication1 sample. If we need to show code then social should get its own sample or linking to this comment on line 78 and providing the code snippet could be adequate:
```
+            app.UseTwitterAuthentication(new TwitterOptions()
 +            {
 +                ConsumerKey = Configuration["Authentication:Twitter:ConsumerId"],
 +                ConsumerSecret = Configuration["Authentication:Twitter:ConsumerSecret"]
 +            });
```